src/neha_Patil/shop.java

```diff
@@ -0,0 +1,106 @@
+package neha_Patil;
+
+public class shop {
+
+
+	int stockofMaggie=00;
+	int stockofDosa=34;
+	int stockofOilPouches=43;
+	int stockofPanipuri=32;
+	int stockofmasala=71;
+
+	void requestedQ(boolean reqType,int maggieq, int dosaQ, int oilQ, int panipuriQ, int masalaQ ){ 
+		if(maggieq==0) 
+			System.out.println("Sorry maggie running out of stock");
```

Comment:
Please change the SOP message, look like it's not giving right information, think on this, if customer pass 0 for maggie packs it does mean we don't have maggie packs right?